src/CalendarPicker.php

```diff
@@ -207,6 +209,10 @@ public function getValue()
 
 	public function getControl()
 	{
+		if ($this->useTime) {
```

Comment:
I think this is too late. When you process data after submit, `getControl` doesn't have to be called.